bucket/gatling.json

```diff
@@ -0,0 +1,43 @@
+{
+    "homepage": "https://gatling.io",
+    "description": "Async Scala-Akka-Netty based Load Test Tool",
+    "version": "3.3.1",
+    "license": "Apache-2.0",
+    "url": "https://repo1.maven.org/maven2/io/gatling/highcharts/gatling-charts-highcharts-bundle/3.3.1/gatling-charts-highcharts-bundle-3.3.1-bundle.zip",
+    "hash": "sha1:6d9630ef1c3af22b217bb7cef3c6179924886469",
+    "extract_dir": "gatling-charts-highcharts-bundle-3.3.1",
+    "bin": [
+        "gatling.bat",
+        "recorder.bat"
+    ],
+    "persist": [
```

Comment:
??

![A](https://i.imgur.com/DbEnH8X.png)

Reply:
I'm a bit worried about persisting THIS conf folder: across the versions of the bundle, the .conf files have changed (adding both files AND entries in each file); what happens if a new version requires a new entry in the gatling.conf file but the previous version's is persisted? It might cause errors.

Reply:
So you are worried, but you still persist it for some reason.

Reply:
Because after my comment I asked a few people who have been using gatling longer than I do and they told me they'd rather run the risk of having to deal with changes in the config structure than having to re-set the personalised configuration after a new version.